src/main/java/apoc/util/DateFormatUtil.java

```diff
@@ -0,0 +1,65 @@
+package apoc.util;
+
+import java.time.format.DateTimeFormatter;
+import java.util.Map;
+
+public class DateFormatUtil {
+
+    public static final Map<String, Object> ISO_DATE_FORMAT = Util.map(
```

Comment:
Perché?
```suggestion
    private static final Map<String, DateTimeFormatter> ISO_DATE_FORMAT = Util.map(
```
aggiungere un metodo `getOrCreate` che ritorna questo:
```
ISO_DATE_FORMAT.containsKey(format.toLowerCase()) ? (DateTimeFormatter) ISO_DATE_FORMAT.get(format.toLowerCase()) : DateTimeFormatter.ofPattern(format);
```

e chiamare questo metodo nei punti opportuni (anche nella classe `TemporalProcedures`)